lib/jemoji.rb

```diff
@@ -3,9 +3,12 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Emoji < Jekyll::Generator
+  class Emoji < Jekyll::Converter
     attr_reader :config
     safe true
+    priority :low
+
+    EMOJIABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
 
```

Comment:
That still feels too manual / heavyweight to maintain. Didn't Jekyll have some "results in HTML?" check? I thought it was called `render`? Maybe I'm mistaken. It seems like knowing whether something is operating on something that'll be HTML is a good method to have, otherwise every plugin maintaining a blacklist is madness.